### ISSUE TITLE
Line awesome class is false !!

### ISSUE THREAD

**Issue by lucrampro** [icons8/line-awesome#54]
Hi all, i open this issue because i use Line awesome with Nuxt.js and i had a probleme.

in Line awesome website  ( https://icons8.com/line-awesome ) for use icon they show you a exemple like this 
`<i class="las la-angle-right"></i>`
 but this DOES NOT WORK !! for use this class you do replace the name class like this
 `<i class="fas fa-angle-right"></i>` 
i dont know why just i want help you :)

**Comment by lucrampro** [comment]
EDIT: I know why class doesn't work !! because in cdn css file (https://maxst.icons8.com/vue-static/landings/line-awesome/font-awesome-line-awesome/css/all.min.css) the name class is not `.la,.lab,.lad,.lal,.lar,.las` but is `.fa,.fab,.fad,.fal,.far,.fas`

**Comment by JustinRvt** [comment]
Hello,
I do believe that .fa,.fab,.fad,.fal,.far,.fas classes belong to Font Awesome, not Line.

**Comment by lucrampro** [comment]
Yes im okay with you but in css file of line awesome the classe's name start with `.fa,.fab,.fad,.fal,.far,.fas`

**Comment by jbovatsek** [comment]
@lucrampro You're using the CDN link that is for converting from FA to LA while not having to update your element classes.   If you're starting a new project from scratch you should use https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css .

Understandable mistake because [the how-to](https://icons8.com/line-awesome/howto) puts the conversion link first.

**Comment by yasiryc** [comment]
I have a similar issue. I am on a new project and using https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css  as CDN. Still, some icons are not loading. for example <i class="las la-eye"></i> is not accessible.

**Comment by fatawesome** [comment]
Hello @yasiryc. Can you please reproduce the issue in a sandbox so we can investigate it?
I've just tested on a small HTML file and it works.
![image](https://user-images.githubusercontent.com/20212361/92697735-4a71dd00-f354-11ea-8cc2-20d28a37bbd7.png)
